src/js/content_scripts/input-mapper-base.js

```diff
@@ -221,6 +221,11 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 funcName: "gamepad.inputMapper.base.clearIntervalRecords",
                 args: ["{that}.intervalRecords"]
             },
+            // TODO: Investigate, identify, and fix tab navigation issues.
```

Comment:
Such as?  Do you already know of any issues?

Reply:
These are a reference to the tab navigation issues you faced and not the browser tab navigation. There's nothing new here, just rearranged the TODO along with the invoker.

Reply:
Got it.